www-apps/hugo/hugo-0.92.2.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module bash-completion-r1
+
+DESCRIPTION="The world's fastest framework for building websites"
+HOMEPAGE="https://gohugo.io https://github.com/gohugoio/hugo"
+
+SRC_URI="
+	https://github.com/gohugoio/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://tastytea.de/files/hugo-0.92.2_deps.tar.xz
+"
+
+# NOTE: To create dependency tarball:
+#       Run `go mod vendor && tar -cJf ${P}_deps.tar.xz vendor`
+
+LICENSE="Apache-2.0 BSD BSD-2 MIT Unlicense"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc +sass"
+
+BDEPEND=">=dev-lang/go-1.16.0"
+RDEPEND="
+	media-libs/libwebp
+	sass? ( dev-libs/libsass )
+"
+DEPEND="${RDEPEND}"
+
+PATCHES=( "${FILESDIR}/0.92.2_link-to-webp-and-sass.patch" )
+
+src_unpack() {
+	unpack ${P}.tar.gz
+	cd "${S}" || die
+	unpack ${P}_deps.tar.xz
+}
+
+src_configure() {
+	export CGO_ENABLED=1
+	export CGO_CFLAGS="${CFLAGS}" \
+		   CGO_CPPFLAGS="${CPPFLAGS} -DLIBWEBP_NO_SRC -DUSE_LIBSASS_SRC" \
+		   CGO_CXXFLAGS="${CXXFLAGS}" \
+		   CGO_LDFLAGS="${LDFLAGS}"
```

Comment:
Hmm, I'm wondering if you couldn't add `-lwebp` here to avoid carrying the patch.

Reply:
Probably, but I'm not sure if it could have side effects if the libwebp wrapper is linked with sass and the libsass wrapper is linked with webp?

Reply:
No clue. I'm not insisting.